### PR TITLE
New post: Fast Pre-trained Model Deployment - The code only approach

### PR DESCRIPTION
**Pull request by mikegc-aws**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [ ] The title of this pull request reflects the title of my content.
- [ ] I have reviewed the [Content Review Checklist](/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [ ] This pull request contains one piece of content (post or page).

**Important: if you are submitting a PR to update existing content, please ensure you [sync'ed your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) first before you created your branch.**

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by mikegc-aws** [comment]
Cyrus review notes: 

excellent post...

2 points.....

1- I would encourage the author to do a part 2 to run an example end-2-end to demonstrate how the the information is used.

2- I would trim the output if the commands

['pytorch-eqa-bert-base-cased',
 'pytorch-eqa-bert-base-multilingual-cased',
 'pytorch-eqa-bert-base-multilingual-uncased',
 'pytorch-eqa-bert-base-uncased',
 'pytorch-eqa-bert-large-cased',
 'pytorch-eqa-bert-large-cased-whole-word-masking',
 'pytorch-eqa-bert-large-cased-whole-word-masking-finetuned-squad',
 'pytorch-eqa-bert-large-uncased',
 'pytorch-eqa-bert-large-uncased-whole-word-masking',
 'pytorch-eqa-bert-large-uncased-whole-word-masking-finetuned-squad',
 'pytorch-eqa-distilbert-base-cased',
 'pytorch-eqa-distilbert-base-multilingual-cased',
 ...
 'pytorch-od1-fasterrcnn-mobilenet-v3-large-320-fpn',
 'pytorch-od1-fasterrcnn-mobilenet-v3-large-fpn',
 'pytorch-od1-fasterrcnn-resnet50-fpn',
 'pytorch-tabtransformerclassification-model',
 'pytorch-tabtransformerregression-model',
 'pytorch-textgeneration1-alexa20b']

something like this....

If one wants to emphasise how many models there are, a line from the len(filtered_models), wouid nicely communicate the post.

These re basically personal choices, and the author is the one to make the decision.

Additionally, putting the outcome of the command in a representative table, and adding a description to the selected models represented in a table, will give good insight to the reader.

Just keeping some of the more popular or useful models and when to use them

Once again, this is how I would do it, and the decision lies with the author

Hope this helps

**Comment by mikegc-aws** [comment]
I will work the feedback into the post.

**Review by lenadroid — APPROVED** [review]
